source/material_model/rheology/frank_kamenetskii.cc

```diff
@@ -81,14 +80,31 @@ namespace aspect
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value.  Units: None");
-
         prm.declare_entry ("Pressure prefactors for Frank Kamenetskii", "0.0",
                            Patterns::List(Patterns::Double (0.)),
                            "A prefactor for the pressure term in the viscosity approximation, "
                            "for a total of N+1 values, where N is the number of all compositional fields or only "
                            "those corresponding to chemical compositions. "
                            "If only one value is given, then all use the same value. "
                            "Units: None");
+        prm.declare_entry ("Reference temperature for Frank Kamenetskii",
```

Comment:
We use plural for parameter names that can contain several values:
```suggestion
        prm.declare_entry ("Reference temperatures for Frank Kamenetskii",
```